public/components/agents/fim/inventory/lib/empty-field-handler.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Wazuh app - Integrity monitoring components
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import { EuiCode, EuiIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React from 'react';
+
+/* This function can be used to render possibly empty fields.
+It takes a render function suitable for an EuiTable and returns another. */
+export const emptyFieldHandler = (renderFn = (value, record) => value) => {
+  return (value, record) => {
```

Comment:
It doesn't work when the value is an object that needs to be mapped, for example in registryValues.tsx
```
    {
      field: 'value',
      name: 'Value name',
      sortable: true,
      render: emptyFieldHandler((item) => item.name),
    },
    {
      field: 'value',
      name: 'Value type',
      sortable: true,
      render: emptyFieldHandler((item) => item.type),
    },
```

Reply:
addressed in [1e90416](https://github.com/wazuh/wazuh-kibana-app/pull/3279/commits/1e904165481426a44912e81766bfd8e7b1dfea73)